modules/mongodb/manifests/s3backup/backup.pp

```diff
@@ -61,7 +57,6 @@
   $private_gpg_key_fingerprint,
   $s3_bucket  = 'govuk-mongodb-backup-s3',
   $server = 'localhost',
-  $standalone  = False,
   $user = 'govuk-backups'
```

Comment:
The `backup::client` class creates a `govuk-backup` user already - could we reuse that instead of creating another user with almost the same name?